vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/promotion/get_available/get_available.cc

```diff
@@ -172,6 +172,14 @@ type::Result GetAvailable::ParseBody(
       promotion->expires_at = time.ToDoubleT();
     }
 
+    auto* claimable_until = item.FindStringKey("claimableUntil");
```

Comment:
Would you mind adjusting the response body description in `get_available.h` and linking to the corresponding server side issue?

Reply:
Thanks for reminding me about the docs in the header file. I've added the field there, but I'd prefer not to add issue links unless it's a TODO thing.

Reply:
Ah, that makes sense :) Done!